packages/db/src/controller/level.ts

```diff
@@ -103,6 +103,28 @@ export class LevelDbController implements DatabaseController<Uint8Array, Uint8Ar
     }
   }
 
+  // https://github.com/Level/abstract-level?tab=readme-ov-file#dbgetmanykeys-options
+  async getMany(keys: Uint8Array[], opts?: DbReqOpts): Promise<KeyValue<Uint8Array, Uint8Array | null>[] | null> {
+    try {
+      this.metrics?.dbReadReq.inc({bucket: opts?.bucketId ?? BUCKET_ID_UNKNOWN}, keys.length);
```

Comment:
```suggestion
      this.metrics?.dbReadReq.inc({bucket: opts?.bucketId ?? BUCKET_ID_UNKNOWN}, 1);
```
This metric tracks # of requests